src/app/shared/services/tour/tour.service.ts

```diff
@@ -45,20 +50,27 @@ export class TourService {
           ) {
             elementSelector = `[data-rowname="${translatedRow.template_component_name}"]`;
           }
+          let stepTitle = this.hackReplaceGlobalInRowMessage(translatedRow.title);
+          let stepContent = this.hackReplaceGlobalInRowMessage(translatedRow.message_text);
+          // if using a template for the step content load using the custom template-container web-component
+          if (translatedRow.message_template) {
+            stepContent = `
+              <web-template-container templatename=${translatedRow.message_template}></web-template-container>
+            `;
+          }
           return {
-            intro: this.hackReplaceGlobalInRowMessage(translatedRow.message_text),
-            title: this.hackReplaceGlobalInRowMessage(translatedRow.title),
```

Comment:
I didn't realise these were already in.

Reply:
Yes, although it is only globals supported. Which is why we still could have added a short term solution to just manually add methods `data` and some others (and still could do), but hopefully being able to embed templates directly is also a good option